lighthouse-core/audits/legacy-javascript.js

```diff
@@ -161,88 +166,98 @@ class LegacyJavascript extends Audit {
   }
 
   static getPolyfillData() {
+    /** @param {string} coreJs2Module */
+    const defineModules = (coreJs2Module) => {
+      return {
+        coreJs2Module,
+        coreJs3Module: coreJs2Module
+          .replace('es6.', 'es.')
```

Comment:
There's a somewhat nice transformation from core js 2 -> core js 3 module names. I'm confident that all of these are accurate–otherwise the `all-legacy-polyfills` variants would error during bundling.